### ISSUE TITLE
install-local-version is platform-dependent

### ISSUE THREAD

**Issue by lognaturel** [getodk/javarosa#654]
#### Software versions 
JavaRosa v3.3.0

#### Problem description
The `install-local-version` script attempts to install to a local mvn repository. However, its version-replacement command doesn't work consistently across platforms.

`sed -i '' '0,/<dependencies>/s/<version>.*<\/version>/<version>local<\/version>/' pom-local.xml`

What it tries to do is `s/<version>.*<\/version>/<version>local<\/version>/'` for the text in `pom.xml` that precedes `<dependencies>`. I believe that should work with GNU sed on Linux but haven't been able to confirm it. It definitely does not work with the `sed` which comes with macOS.

#### Steps to reproduce the problem
Run `./install-local-version.sh`

#### Expected behavior
Local mvn install

#### Other information 
I propose we drop the script and just document manually setting the versions to `local` and running the `mvn` command. We don't do this often and I think automating it in a platform-agnostic way is going to be a real pain.

**Comment by seadowg** [comment]
It's so annoying that there's no way (that I've found anyway) to override the version in the `pom.xml` when doing an `mvn install`.

It would be nice to have this for iterating, personally I found it really useful when it worked. Is there a simpler tool we could use to replace the version? It seems earth shatteringly simple, but I guess it might not work without a proper scripting language - potentially we could write a Gradle task to do this?

**Comment by lognaturel** [comment]
Yeah, a gradle task seems promising. But honestly changing the versions to local and not committing that has worked fine for me. Not sure it would really save time to automate at this point.

**Comment by seadowg** [comment]
> Not sure it would really save time to automate at this point.

I guess it's "automation as documentation" over "automation to optimize time taken" for me here. I'd be interested to have a stab at the Gradle task at some point so happy to take that on!
